lua/coq.lua

```diff
@@ -1,157 +1,129 @@
-      local win_proxy = cwd .. [[/win.bat]]
+local main = function()
+    local v_py = cwd .. (is_win and [[/.vars/runtime/Scripts/python.exe]] or
+                   '/.vars/runtime/bin/python3')
+    local win_proxy = cwd .. [[/win.bat]]
 
-      if is_win then
-        if vim.fn.filereadable(v_py) == 1 then
-          return {v_py}
+    if is_win then
+        if v.fn.filereadable(v_py) == 1 then
+            return {v_py}
         else
-          return {win_proxy, py3}
+            return {win_proxy, py3}
         end
-      else
-        if vim.fn.filereadable(v_py) == 1 then
-          return {v_py}
+    else
+        if v.fn.filereadable(v_py) == 1 then
+            return {v_py}
         else
-          return {py3}
+            return {py3}
         end
-      end
     end
+end
 
-    local start = function(deps, ...)
-      local args =
-        vim.tbl_flatten {
-        deps and py3 or main(),
-        {"-m", "coq"},
-        {...}
-      }
+local start = function(deps, ...)
+    local args = v.tbl_flatten {deps and py3 or main(), {'-m', 'coq'}, {...}}
 
-      local params = {
+    local params = {
         cwd = cwd,
         on_exit = on_exit,
         on_stdout = on_stdout,
-        on_stderr = on_stderr
-      }
-      job_id = vim.fn.jobstart(args, params)
-      return job_id
-    end
+        on_stderr = on_stderr,
+    }
+    job_id = v.fn.jobstart(args, params)
+    return job_id
+end
 
-    coq.COQdeps = function()
-      start(true, "deps")
-    end
+M.COQdeps = function() start(true, 'deps') end
 
-    vim.api.nvim_command [[command! -nargs=0 COQdeps lua coq.COQdeps()]]
+api.nvim_command [[command! -nargs=0 COQdeps lua M.COQdeps()]]
 
-    local set_coq_call = function(cmd)
-      coq[cmd] = function(...)
+local set_coq_call = function(cmd)
+    M[cmd] = function(...)
         local args = {...}
 
         if not job_id then
-          local server = vim.api.nvim_call_function("serverstart", {})
-          job_id = start(false, "run", "--socket", server)
+            local server = api.nvim_call_function('serverstart', {})
+            job_id = start(false, 'run', '--socket', server)
         end
 
         if not err_exit and _G[cmd] then
-          _G[cmd](args)
+            _G[cmd](args)
         else
-          vim.defer_fn(
-            function()
-              if err_exit then
-                return
-              else
-                coq[cmd](unpack(args))
-              end
-            end,
-            POLLING_RATE
-          )
+            v.defer_fn(function()
+                if err_exit then
+                    return
+                else
+                    M[cmd](unpack(args))
+                end
+            end, POLLING_RATE)
         end
-      end
     end
+end
 
-    set_coq_call("COQnow")
-    vim.api.nvim_command [[command! -nargs=* COQnow lua coq.COQnow(<f-args>)]]
+set_coq_call('COQnow')
+api.nvim_command [[command! -nargs=* COQnow lua M.COQnow(<f-args>)]]
```

Comment:
shouldn't it be 
`require'coq'.COQnow`
instead of `M`

Reply:
Shit. Almost broke it. It should be require ("coq"). function.

Reply:
Command issue is fixed and tested.